apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet.IntegrationTests/ErrorReporting/ErrorReportingTest.cs

```diff
@@ -125,36 +125,48 @@ public class ErrorReportingTest
                 Assert.Equal(HttpStatusCode.InternalServerError, response.StatusCode);
 
                 var errorEvent = _polling.GetEvents(startTime, testId, 1).Single();
-                VerifyAutomaticLogging(errorEvent, testId, "SendAsync");
+                VerifyFullLogging(errorEvent, testId, "SendAsync");
             }
         }
 
         [Fact]
-        public async Task ManualLog()
+        public async Task ManualLog_GoogleLogger()
         {
             string testId = Utils.GetTestId();
             DateTime startTime = DateTime.UtcNow;
 
             using (TestServer server = TestServer.Create<ErrorReportingTestApplication>())
             using (var client = server.HttpClient)
             {
-                var response = await client.GetAsync($"api/ErrorReporting/ThrowCatchLog/{testId}");
+                var response = await client.GetAsync($"api/ErrorReporting/ThrowCatchWithGoogleLogger/{testId}");
```

Comment:
Possibly use `{nameof(ThrowCatchWithGoogleLogger)}` rather than hard-coding the literal? That would be more refactoring-friendly, assuming the method name is the important part here.

Reply:
Done here and everywhere.